pages/index.vue

```diff
@@ -167,12 +167,28 @@
         </h1>
       </div>
     </section>
+    <!-- changes are made -->
+    <section>
+      <div class="flex flex-col md:flex-row m-auto w-5/6 text-center  ">
+        <div class="m-4 shadow-lg overflow-hidden pb-4 rounded-b-2xl  flex flex-col content-center items-center justify-center ">
+          <img src="https://images.pexels.com/photos/265216/pexels-photo-265216.jpeg" alt="crop image " srcset="">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Eum nihil quos corrupti, quisquam fugiat libero nisi assumenda qui optio modi?
+          <button type="submit" class="bg-green-400 hover:bg-green-300 text-white font-bold py-1 px-4 border-b-4 border-green-600 hover:border-green-400  rounded mt-2" >More..</button>
```

Comment:
change the background color from `bg-green-400` to `bg-green-100` to be more consistent with the brand design system. Also, add some transitions on the hover _e.g._ `ease-in-out duration-300`